install.sh

```diff
@@ -0,0 +1,198 @@
+    fi
+    if [ ! command -v pip3 &> /dev/null ]; then
+        sudo apt-get install python3-pip
+    else
+        echo "pip is already installed"
+    fi
+    if [ ! command -v git &> /dev/null ]; then
+        sudo apt-get install git
+    else
+        echo "git is already installed"
+    fi
+}
+
+function install_fedora(){
+    echo "Installing python"
+    if [ ! command -v python3 &> /dev/null ]; then
+        sudo dnf install python3
+    else
+        echo "Python is already installed"
+    fi
+    if [ ! command -v pip3 &> /dev/null ]; then
+        sudo dnf install python3-pip
+    else
+        echo "pip is already installed"
+    fi
+    if [ ! command -v git &> /dev/null ]; then
+        sudo dnf install git
+    else
+        echo "git is already installed"
+    fi
+}
+
+function install_centos(){
+    echo "Installing python"
+    if [ ! command -v python3 &> /dev/null ]; then
+        sudo yum install python3
+    else
+        echo "Python is already installed"
+    fi
+    if [ ! command -v pip3 &> /dev/null ]; then
+        sudo yum install python3-pip
+    else
+        echo "pip is already installed"
+    fi
+    if [ ! command -v git &> /dev/null ]; then
+        sudo yum install git
+    else
+        echo "git is already installed"
+    fi
+}
+
+function get_the_bot(){
+    echo "Downloading the bot"
+    git clone https://github.com/elebumm/RedditVideoMakerBot.git
+}
+
+function install_python_dep(){
+    echo "Installing python dependencies"
+    cd RedditVideoMakerBot
+    pip3 install -r requirements.txt
+}
+
+function install_playwright(){
+    echo "Installing playwright"
+    cd RedditVideoMakerBot
+    python3 -m playwright install
+    python3 -m playwright install-deps
+}
+
+install_deps(){
+    if [ "$(uname)" == "Darwin" ]; then
+        install_macos || install_fail
+    elif [ "$(uname)" == "Linux" ]; then
+        if [ "$(cat /etc/os-release | grep -i "arch")" ]; then
+            install_arch || install_fail
+        elif [ "$(cat /etc/os-release | grep -i "debian")" ]; then
+            install_deb || install_fail
+        elif [ "$(cat /etc/os-release | grep -i "fedora")" ]; then
+            install_fedora || install_fail
+        elif [ "$(cat /etc/os-release | grep -i "centos")" ]; then
+            install_centos || install_fail
+        else
+            printf "Your OS is not supported\n Please install python3, pip3 and git manually\n After that, run the script again with the -p option to install python and playwright dependencies\n If you want to add support for your OS, please open a pull request on github\n https://github.com/elebumm/RedditVideoMakerBot"
+        fi
+    fi
+}
+
+
+function install_main(){
+    echo "Installing dependencies..."
+    if [[ ASSUME_YES -eq 1 ]]; then
+        echo "Assuming yes"
+    else
+        echo "Continue? (y/n)"
+        read answer
+        if [ "$answer" != "y" ]; then
+            echo "Aborting"
+            exit 1
+        fi
```

Comment:
What do you think about using enter instead of y/n? 
It simple and fast) 
```suggestion
        echo "Continue? (y/n)"
        read -r -s -n 1 answer
        if [[ $answer != "" ]]; then
            echo "Aborting"
            exit 1
        fi
```